packages/vite-node/src/utils.ts

```diff
@@ -19,7 +19,7 @@ export function normalizeId(id: string, base?: string): string {
     .replace(/^(node|file):/, '')
     .replace(/^\/+/, '/') // remove duplicate leading slashes
     .replace(/[?&]v=\w+/, '?') // remove ?v= query
-    .replace(/\?import/, '') // remove ?import query
+    .replace(/[?&]import/, '?') // remove ?import query
```

Comment:
Maybe it would be better to replace `?import` by `?` and `&import` by `&`? I don't know if having two `?` in the URL will end up confusing Vite 🤔 
```suggestion
    .replace(/([&?])(import)/,'$1') // remove ?import query
```